backend/scoreserver/domain/fixture.go

```diff
@@ -137,6 +137,46 @@ func FixDescriptiveProblem1(tb testing.TB, data *DescriptiveProblemData) *Descri
 	return problem
 }
 
+func FixNotice1(tb testing.TB, data *NoticeData) *Notice {
+	tb.Helper()
+
+	if data != nil {
+		tb.Fatal("additional data is not supported")
+	}
+
+	now := time.Now()
+	yesterday := now.Add(-24 * time.Hour)
+	tomorrow := now.Add(24 * time.Hour)
```

Comment:
fixture なのでマジックナンバーがあるのは問題なくて，nolint してあげればよさそう
```suggestion
	tomorrow := now.Add(24 * time.Hour) //nolint:mnd // fixture
```